src/pages/Contributors/index.js

```diff
@@ -170,9 +171,9 @@ export default function Contributors() {
     setLoading(true);
     const affiliated = [];
     const unaffiliated = [];
-    const input = searchQuery.toLowerCase().replace(/\s/g, '');
+    const input = searchQuery.toLowerCase().replace(/[%250\s]/g, '');
     for (const org of organizations) {
-      const orgName = org.name.toLowerCase().replace(/\s/g, '');
+      const orgName = org.name.toLowerCase().replace(/[%250\s]/g, '');
```

Comment:
Do we still need to find and replace uri encoded spaces if they'll be decoded by decodeURIComponent?

Reply:
No, but the regex was implemented before UriDecode, should I remove?

Reply:
probably a good idea to remove it so we have less code to maintain.  thanks